pinot-segment-local/src/main/java/org/apache/pinot/segment/local/aggregator/ValueAggregatorFactory.java

```diff
@@ -66,6 +67,11 @@ public static ValueAggregator getValueAggregator(AggregationFunctionType aggrega
       case DISTINCTCOUNTTHETASKETCH:
       case DISTINCTCOUNTRAWTHETASKETCH:
         return new DistinctCountThetaSketchValueAggregator();
+      case DISTINCTCOUNTTUPLESKETCH:
+      case DISTINCTCOUNTRAWINTEGERSUMTUPLESKETCH:
+      case AVGVALUEINTEGERSUMTUPLESKETCH:
+      case SUMVALUESINTEGERSUMTUPLESKETCH:
+        return new IntegerTupleSketchValueAggregator(IntegerSummary.Mode.Sum);
```

Comment:
For AVGVALUEINTEGERSUMTUPLESKETCH , is the plan to support 'AvgValue' mode, later ? Looks like IntegerSummary does not support Avg now.